circle.yml

```diff
@@ -45,6 +45,9 @@ checkout:
 
 dependencies:
   pre:
+    # CircleCI v1 caching doesn't work for parallel builds
+    # Clean any possible cache from the previous runs to avoid re-using wrong distro cache
+    - rm -rf node_modules || true
```

Comment:
CircleCI caches `node_modules` from the `0`th parallel container https://circleci.com/docs/1.0/how-cache-works/#caching-in-parallel-builds
resulting in other distros reusing that and linked to a wrong shared libraries like `EL6` links to `U14` libs.

It's horrible.